core/deployment/src/main/java/io/quarkus/deployment/ExtensionLoader.java

```diff
@@ -183,9 +201,11 @@ private static boolean isRecorder(AnnotatedElement element) {
                 reader.getBuildTimePatternMap());
         final DefaultValuesConfigurationSource ds2 = new DefaultValuesConfigurationSource(
                 reader.getBuildTimeRunTimePatternMap());
+        final PropertiesConfigSource platformConfigSource = new PropertiesConfigSource(platformProperties, "Quarkus platform",
+                ConfigSource.DEFAULT_ORDINAL);
```

Comment:
Since these would be default values, they should have a low ordinal.  The constant `DEFAULT_ORDINAL` is the default ordinal for config sources, not the ordinal for _default config sources_, in case that was a point of confusion.  Using the default ordinal means that the defaults may take precedence over some explicitly configured values.

Our default value configuration source is using an ordinal of `Integer.MIN_VALUE` (see `io.quarkus.deployment.configuration.DefaultValuesConfigurationSource`), so I think it would be appropriate to use, say, `Integer.MIN_VALUE + 1000` or something along those lines.